acquire-video-runtime/src/runtime/source.c

```diff
@@ -66,6 +60,9 @@ video_source_thread(struct video_source_s* self)
 
         size_t sz = bytes_of_image(&info.shape);
         size_t nbytes = sizeof(struct VideoFrame) + sz;
+        const size_t align = 8;
+        const size_t padding = (align - (nbytes & (align - 1))) & (align - 1);
+        const size_t nbytes_aligned = nbytes + padding;
```

Comment:
IMO, this is more readable if you just rely on the `/` to truncate the fractional part. And I think it's ok to substitute in `align` - just make a comment about the constants.

```suggestion
        // padding to 8-byte aligned size
        const size_t nbytes_aligned = 8*((nbytes+7)/8);
```